iogt/settings/base.py

```diff
@@ -422,3 +422,4 @@
 from iogt.patch import *
 
 WAGTAILTRANSFER_UPDATE_RELATED_MODELS = ['wagtailimages.image', 'wagtailsvg.svg',]
+SHOW_ERROR_FOR_REFERENCED_PAGES = True
```

Comment:
Should this be prefixed with `WAGTAILTRANSFER`?

Reply:
done